tools/gen_sig_info.py

```diff
@@ -227,6 +227,7 @@ def extract_sig_info(sig_info, extra_settings=None, extra_cflags=None):
   settings = {
     'USE_PTHREADS': 1,
     'STACK_OVERFLOW_CHECK': 1,
+    'MAX_WEBGL_VERSION': 2,
```

Comment:
No signatures change with this setting, correct?

Reply:
No.  If they did, then you would see the generated file change.